### PR TITLE
feat: sentry integration

### PR DESCRIPTION
**Pull request by dexturr**
## Chore Sentry (Frontend)

Integrate Sentry with the frontend on the global level. There are still cases where errors will be missed but I will do that in a follow up PR to keep this small.

### CODE REVIEW

**Review by mattrussell36 — APPROVED** [review]

**Comment by mattrussell36** [comment]
Only thing I'd add is perhaps exposing `captureMessage`, `captureException` from the hook or some other utility function so you don't clutter the app with Sentry and create a hard to change dependency (like we currently have with the mono repo)

File: frontend/routes/home/index.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import { Navigate } from 'react-router-dom'
 import { useGetRedirectPath } from '../../hooks/redirect-path'
+import { useSentry } from '../../hooks/sentry'
 
 export const Home = () => {
   const { loading, error, path } = useGetRedirectPath()
+  useSentry()
```
Comment (edd) [inline frontend/routes/home/index.tsx:7]: **question (non-blocking)**:  I feel like this means that telemetry defaults to on? Is that alright?

**Review by edd — CHANGES_REQUESTED** [review]
console.log & a question

File: frontend/routes/home/index.tsx (hunk 1 above)
Comment (dexturr) [reply frontend/routes/home/index.tsx:7]: Nope, use sentry checks the settings and only enables if the setting has been saved by the user

Comment (dexturr) [reply frontend/routes/home/index.tsx:7]: https://github.com/vegaprotocol/vegawallet-browser/pull/317/files#diff-2a5e93a24268b8525fdd0faa490d14140966605c64c4ff868c3daea3c3e5cc90R29

**Review by edd — APPROVED** [review]

**Pull request merged**